tests/test_image.py

```diff
@@ -29,24 +29,50 @@ def tearDown(self):
         pass
 
     def testImShift(self):
```

Comment:
While we are in here, can we parametrize this to run for odd as well.

Reply:
you mean odd resolution image right?

Reply:
Yep